app/frontend/src/components/admin/StatusTable.vue

```diff
@@ -0,0 +1,139 @@
+            color="secondary"
+          >close</v-icon>
+          <v-icon v-else color="secondary">mdi-help-circle</v-icon>
+        </template>
+
+        <template v-slot:item.inspectorName="{ item }">{{ item.inspectorName }}</template>
+
+        <template v-slot:item.inspectionDate="{ item }">{{ formatDate(item.inspectionDate) }}</template>
+      </v-data-table>
+    </v-container>
+  </v-card>
+</template>
+
+<script>
+import ipcService from '@/services/ipcService';
+
+export default {
+  name: 'SubmissionsTable',
+  computed: {
+    responsiveCell() {
+      return this.$vuetify.breakpoint.name == 'xs'
+        ? 'v-data-table__mobile-table-row'
+        : '';
+    }
+  },
+  data() {
+    return {
+      search: '',
+      headers: [
+        { text: 'Status', value: 'status' },
+        { text: 'Date', align: 'start', value: 'createdAt' },
+        { text: 'Grade', align: 'start', value: 'grade' },
+        { text: 'Assigned Inspector', value: 'inspectorName' },
+        { text: 'Inspection Date', value: 'inspectionDate' }
+      ],
+      statuses: [],
+      ipcPlanId: this.$route.params.ipcPlanId,
+      loading: true,
+      showAlert: false,
+      alertType: null,
+      alertMessage: '',
+      expanded: []
+    };
+  },
+  methods: {
+    formatDate(date) {
+      return date ? new Date(date).toLocaleString() : '';
+    },
+    getData() {
+      ipcService
+        .getIPCContent(this.ipcPlanId)
+        .then(response => {
+          this.statuses = response.data.inspectionStatuses;
+          if (!this.statuses.length) {
+            this.showTableAlert('error', 'Inspection statuses not found');
+          }
+        })
+        .catch(() => {
+          this.showTableAlert('error', 'No response from server');
+        })
+        .finally(() => {
+          this.loading = false;
+        });
+    },
+    showTableAlert(typ, msg) {
+      this.showAlert = true;
+      this.alertType = typ;
+      this.alertMessage = msg;
+      this.loading = false;
+    }
+  },
+  mounted() {
+    this.getData();
+  }
+};
+</script>
+
+<style scoped>
+.status-search {
+  width: 100%;
+  max-width: 20em;
+  float: right;
+}
+.status-table {
+  clear: both;
+}
+.status-table >>> tr.v-data-table__expanded__row td {
+  border-bottom: 0 !important;
+}
+.status-table >>> tr.v-data-table__expanded__content {
+  -webkit-box-shadow: none !important;
+  box-shadow: none !important;
+}
+.status-table >>> tr.v-data-table__expanded__content td {
+  padding-bottom: 1em;
+}
+div.ipc-expanded {
+  font-size: 85% !important;
+  color: #494949 !important;
+  padding: 1rem 0;
+}
+/* mobile view */
+tr.v-data-table__expanded__content
+  td.v-data-table__mobile-table-row:nth-child(1) {
+  display: none !important;
+}
+tr.v-data-table__expanded__content
+  td.v-data-table__mobile-table-row:not(:nth-child(1)) {
+  padding: 0;
+}
+td.v-data-table__mobile-table-row div.ipc-expanded {
+  padding: 0.2rem 1rem;
+}
```

Comment:
Same as the table of submissions? If so would be good to see if it can be extracted out to an scss import, or global style, or something else...

Reply:
Yea I was thinking the same. For now I'll just leave this as is because it'll achieve what we want, but definitely should be revisited to reduce CSS duplication.